packages/js/client/src/__tests__/e2e/wasm-rs.spec.ts

```diff
@@ -330,3 +335,95 @@ describe("wasm-rs test cases", () => {
     );
   })
 });
+
+describe.skip("Wasm-rs benchmarking", () => {
```

Comment:
We can merge this as `skip`ped for now, but we should probably make a benchmarking project that does this in the near future.